src/main/java/jenkins/scm/impl/trait/WildcardSCMHeadFilterTrait.java

```diff
@@ -129,6 +130,7 @@ private String getPattern(String branches) {
     /**
      * Our descriptor.
      */
+    @Symbol("headsWildcardFilter")
```

Comment:
`headWildcardFilter` by analogy with `headRegexFilter` and following the general English rule (as I understand it) that this usage of “heads” is a kind of unmarked genitive